### PR TITLE
Lambda handler

### PR DESCRIPTION
**Pull request by joe1606**
Created a lambda handler function

### CODE REVIEW

File: pipeline/handler.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""Lambda Handler"""
+
+import traceback
+from os import environ as ENV
+from dotenv import load_dotenv
```
Comment (laUmar123) [inline pipeline/handler.py:5]: you're not using these so you don't need to import them

Comment (joe1606) [reply pipeline/handler.py:5]: Good catch

**Comment by joe1606** [comment]
closes #69

**Review by laUmar123 — APPROVED** [review]
Nice work

**Pull request merged**